templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Manufacturer Point</h1>
+  <p>Are you sure you wand to delete the Manufacturer Point: {{ manufacturer }}</p>
+  <p><i> All Cars with this Point will be also deleted! </i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
```

Comment:
Code Style issue: It's recommended to add a `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.

Reply:
done